### PR TITLE
[MPICore] fix file path

### PR DESCRIPTION
**Pull request by philbucher**
missing `GetFilePath`

### CODE REVIEW

File: kratos/mpi/tests/test_mpi_communicator.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 import KratosMultiphysics.mpi as KratosMPI
-import KratosMultiphysics.MetisApplication as KratosMetis
+import KratosMultiphysics.MetisApplication as KratosMetis # TODO refactor this by using the "distributed_import_model_part_utility"
 import KratosMultiphysics.kratos_utilities as kratos_utilities
 
+import os
 
 def GetFilePath(fileName):
     return os.path.dirname(os.path.realpath(__file__)) + "/" + fileName
```
Comment (sunethwarna) [inline kratos/mpi/tests/test_mpi_communicator.py]:
```suggestion
    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
```

Comment (sunethwarna) [reply kratos/mpi/tests/test_mpi_communicator.py]: This is not relevant to this PR, but this makes it cleaner i guess...

**Review by sunethwarna — DISMISSED** [review]
Minor suggestion only :)

File: kratos/mpi/tests/test_mpi_communicator.py (hunk 1 above)
Comment (philbucher) [reply kratos/mpi/tests/test_mpi_communicator.py]: you are right
too much copy-paste...
thx for paying attention :)

**Review by sunethwarna — APPROVED** [review]

**Pull request merged**